uwblueprint/focus-on-nature
# feat: add delete waitlisted camper api

## Notion ticket link

[Ticket Name](https://www.notion.so/uwblueprintexecs/Task-Board-db95cd7b93f245f78ee85e3a8a6a316d)



## Implementation description
* Created delete waitlisted campers endpoint
* Implementation involves deleting waitlisted camper and its constituents and sending a confirmation email to parent who requested it



## Steps to test
1. Register a waitlisted camper (use endpoint)
2. Delete waitlisted camper
3. Ensure mongodb is cleared and a confirmation email is sent



## What should reviewers focus on?
* 


## Checklist
